plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/AbstractTestDeltaLakeCreateTableStatistics.java

```diff
@@ -445,8 +445,8 @@ public void testMultiFileTable()
         try (TestTable table = new TestTable(
                 "test_partitioned_table_",
                 ImmutableList.of(),
-                ImmutableList.of(),
-                "SELECT name FROM tpch.tiny.nation UNION select name from tpch.tiny.customer",
+                ImmutableList.of("partitionkey"),
+                "SELECT 'nation' AS partitionkey, name FROM tpch.tiny.nation UNION SELECT 'customer' AS partitionkey, name FROM tpch.tiny.customer",
```

Comment:
I assume UNION (should be UNION ALL, really) was used to ensure multiple data files.

should we drop the second UNION branch now?